src/app/modules/group/components/group-join-by-code/group-join-by-code.component.html

```diff
@@ -66,7 +66,7 @@
             class="submit-duration-button"
             (click)="codeLifetimeControlValue && submitCodeLifetime(codeLifetimeControlValue.ms)"
             [disabled]="processing || !codeLifetimeControlValue || !control.dirty || codeLifetimeControlValue.ms === group.codeLifetime?.ms"
-            i18n-pTooltip pTooltip="This code will expire after the given duration (reset current expiration)"
+            [pTooltip]="durationTooltip"
```

Comment:
Is there a reason not to use the i18n metadata via attribute `i18n-pTooltip="@@resetCurrentExpiration"` ?

Reply:
Comment after talking with @Iloveall: Because the translation is split into 2 parts and the attribute does not allow translation splitting